app/models/topic_timer.rb

```diff
@@ -20,6 +22,7 @@ class TopicTimer < ActiveRecord::Base
   validates :category_id, presence: true, if: :publishing_to_category?
 
   validate :executed_at_in_future?
+  validate :duration_ok?
```

Comment:
maybe duration_in_range ? that _ok? feels weird to me, kinda redundant with validate too. It s the purpose of validate to check if "ok"

Reply:
Fixed!